erpnext/schools/doctype/student_group_creation_tool/student_group_creation_tool.py

```diff
@@ -67,6 +67,49 @@ def create_student_groups(self):
 			student_group.max_strength = d.max_strength
 			student_group.academic_term = self.academic_term
 			student_group.academic_year = self.academic_year
+			student_list = get_students(self.academic_year, d.group_based_on, self.academic_term, self.program, d.batch, d.course)
+			if student_list:
+				for student in student_list:
+					student_group.append('students', student)
 			student_group.save()
 
 		frappe.msgprint(_("{0} Student Groups created.".format(l)))
+
+@frappe.whitelist()
```

Comment:
@Zlash65 Please don't write the duplicated code. You can import the function from the student group.